multichain-testing/tools/sleep.ts

```diff
@@ -28,11 +28,11 @@ const retryUntilCondition = async <T>(
   {
     maxRetries = 6,
     retryIntervalMs = 3500,
-    log = () => {},
+    log = console.log,
```

Comment:
personally I find  this log output noisy. Good to change `console.log` below to this `log`, but consider leaving the noop default for silence

Reply:
This (+ `console.log -> log`) would impact a lot of existing tests, so I chose to preserve the existing behavior. I'll see how this looks in local runs and will look to land it in the future:
```diff
+    log = () => {},
-    log = console.log,
```

Reply:
s/impact/denoise 😉